presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSnippetSuite.scala

```diff
@@ -365,18 +367,33 @@ class CompletionSnippetSuite extends BaseCompletionSuite:
     checkSnippet(
       s"""|package example
           |
-          |object Widget{}
+          |object TestObject {}
           |object Main {
-          |  Wi@@
+          |  TestObjec@@
           |}
           |""".stripMargin,
-      """|Widget -  example
-         |Window -  java.awt
-         |WindowPeer -  java.awt.peer
-         |WithFilter -  scala.collection
+      """|TestObject -  example
+         |""".stripMargin,
+      includeDetail = true,
+    )
+
+  @Test def `dont-enter-empty-paramlist` =
+    checkSnippet(
+      s"""|package example
+          |
+          |object Main {
+          |  ListMa@@
+          |}
+          |""".stripMargin,
+      """|ListMap -  scala.collection.immutable
+         |ListMap -  scala.collection.mutable
+         |ListMap($0) - [K, V](elems: (K, V)*): ListMap[K, V]
+         |new ListMap - [K, V]: ListMap[K, V]
+         |ListMapBuilder - [K, V]: ListMapBuilder[K, V]
```

Comment:
I don't agree that we should add parentheses when they were not present in original definition. Especially that there is a difference in Scala 3 between having and not having parentheses

Reply:
Okay, agreed. Though that's a bit tricky, since:
```scala
class A[T] { val f = 1 }
class B { val f = 1 }

object O {
  val _ = A[Int].f    // valid
  val _ = A().f       // valid
  val _ = A.f         // not valid
  val _ = B.f         // not valid
  val _ = (new B).f   // valid (xd)
}
```
But that's not important.

On a different note `ListMapBuilder` is `private[immutable]`, so probably shouldn't be shown at all.